shared/chat/common/avatars.js

```diff
@@ -1,8 +1,8 @@
 // @flow
 import * as React from 'react'
 import shallowEqual from 'shallowequal'
```

Comment:
i don't think we need 'common' if its used by multiple things it can just go up a level

Reply:
So for this, should it be at `chat/avatars.js`? It's used in both `chat/inbox` and `chat/conversation-list`